packages/@postgrestools/postgrestools/scripts/generate-packages.mjs

```diff
@@ -219,6 +219,35 @@ function copySchemaToNativePackage(platform, arch) {
 	fs.chmodSync(schemaTarget, 0o666);
 }
 
+function copyReadmeToPackage(packagePath) {
+	const packageRoot = resolve(PACKAGES_POSTGRESTOOLS_ROOT, packagePath);
+	const readmeSrc = resolve(POSTGRESTOOLS_ROOT, "README.md");
```

Comment:
do we need to adjust the `README.md`? there are probably some `postgrestools` mentions in there, right?

Reply:
nope, only in the link to the vscode extension